src/jitlog/build.lua

```diff
@@ -1,10 +1,9 @@
-local stdout = io.stdout
-local args = {...}
-
-GC64 = args[1] == "GC64"
+local stdout = io.stdout
```

Comment:
Is it worth making this a local variable given that it's only referenced once below?

Reply:
its used 5 times below

Reply:
OK.